packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/UploadControls.tsx

```diff
@@ -124,6 +128,7 @@ export const UploadControls = (): JSX.Element => {
       >
         Start upload
       </Primary>
+      <Destination>{history[history.length - 1]}</Destination>
```

Comment:
Should we only show the last folder or should we show the whole path for nested locations?

Currently this folder also includes a trailing slash as seen in the PR description. Should it be removed so it's only the folder name?